spec.md

```diff
@@ -1427,20 +1585,17 @@ A register is initialized with an indeterminate value (see
 
 ## Invalidates
 
-An invalidate statement is used to indicate that a circuit component contains
-indeterminate values (see [@sec:indeterminate-values]). It is specified as
-follows:
+An invalidate statement is used to indicate that a circuit component contains indeterminate values (see [@sec:indeterminate-values]).
```

Comment:
it seems this is purely a whitespace change? This doc has been formatted for 80 character limit, is there a reason you are changing this line?

Reply:
It looks like I removed the second sentence. (And this was outdated by another change, so just keeping that in mind).

I would like to gradually shift over to using a git-friendly formatting, with one(-ish) sentence per line.

Reply:
I'll also ensure I have a reasonable line length. I just realized my editor settings don't make it obvious for me to tell when I hit the 80 or 100 col line limit.

Reply:
I pushed a change to cap all of my changes at 100 cols.

Hopefully that isn't controversial.